test/xpu_api/utilities/pairs/pairs.pair/noexcept_move_assign_g_pairs.pass.cpp

```diff
@@ -0,0 +1,57 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+#include <oneapi/dpl/utility>
+
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+
+struct TestStruct
+{
+};
+
+void
+kernel_test()
+{
+    sycl::queue deviceQueue = TestUtils::get_test_queue();
+    {
+        deviceQueue.submit([&](sycl::handler& cgh) {
+            cgh.single_task<class KernelTest>([=]() {
+                typedef dpl::pair<int, int> tt1;
+                typedef dpl::pair<int, float> tt2;
+                typedef dpl::pair<TestStruct, TestStruct> tt3;
```

Comment:
Same as for tuples - we need a test with one argument which is not nothrow move assignable.

Reply:
Done